docs/easing.md

```diff
@@ -234,7 +234,7 @@ A useful tool to visualize cubic bezier curves can be found at http://cubic-bezi
 ### `in()`
 
 ```javascript
-static in(easing);
+static in easing;
```

Comment:
This looks wrong?

Reply:
Yeah it does that for some reason. I'll manually revert.